modules/cnf-cluster-instance-cr-conditions-reference.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/installing_with_siteconfig_operator/cnf-understanding-siteconfig-operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cnf-clusterinstance-conditions-reference_{context}"]
+= ClusterInstance CR conditions
+
+The samples resource maintains the following conditions in its status:
```

Comment:
This section needs to be introduced a bit better - it feels a bit disconnected from the other section. A natural flow is missing...